README.md

```diff
@@ -180,14 +200,19 @@ import com.rudderstack.generated.*
 
 ```javascript
 // Import RudderStack JS SDK and initialize it
-const rudderanalytics = require('rudder-sdk-js');
-rudderanalytics.load(YOUR_WRITE_KEY, DATA_PLANE_URL);
+var RudderAnalytics = require('@rudderstack/analytics-js');
```

Comment:
```suggestion
const RudderAnalytics = require('@rudderstack/analytics-js');
```

Reply:
I'll let @MoumitaM address this in her branch.